common/src/main/java/com/wynntils/models/objectives/WynnObjective.java

```diff
@@ -23,6 +23,9 @@ public final class WynnObjective {
     private final boolean isGuildObjective;
     private final boolean hasEventBonus;
 
+    public static WynnObjective DEMO_GUILD = new WynnObjective(
+            "Guild Objective", new CappedValue(5, 15), 0, StyledText.fromString("Guild Objectives"), true, false);
```

Comment:
```suggestion
            "Guild Objective", new CappedValue(5, 15), 0, StyledText.fromString("Slay Loot Chests"), true, false);
```
:)

I like it when we have a bit more humorous examples in the preview. But if not, they should be more realistic. "Guild Objectives" sound like a title, not an actual objective.